src/remoteConnector.ts

```diff
@@ -565,6 +565,7 @@ export default class RemoteConnector extends Disposable {
 			}).connect({
 				host: sshDestInfo.hostName,
 				username: sshDestInfo.user,
+				readyTimeout: 40000,
```

Comment:
What is it seconds?

Reply:
Is it possible to show a process bar when connecting and show "fallback to/try with local-app" action after 20 seconds? 40s or 1-2minutes are too long to wait

Reply:
> Is it possible to show a process bar when connecting

yeah possible but this connection does nothing it's just a quick test to check if port is not blocked so not sure if we should expose it to the user

Reply:
> Is it possible to show a process bar when connecting and show "fallback to/try with local-app" action after 20 seconds? 40s or 1-2minutes are too long to wait

I mean generally it should be very quick, so maybe it is not bad to show progress bar if you really have problems, then we need to add some messaging `Verifying SSH connection...`